db/system_distributed_keyspace.hh

```diff
@@ -48,10 +48,10 @@ public:
     static constexpr auto VIEW_BUILD_STATUS = "view_build_status";
 
     /* Nodes use this table to communicate new CDC stream generations to other nodes. */
-    static constexpr auto CDC_TOPOLOGY_DESCRIPTION = "cdc_topology_description";
+    static constexpr auto CDC_TOPOLOGY_DESCRIPTION = "cdc_generations";
```

Comment:
I would have changed the name of the variable as well. No reason to leave confusing riddles for developers a year from now that have to figure out why the name of the table and the variable is different.
But seeing @psarna 's comment that we should commit this quickly, I'll commit this as-is but please consider sending a followup patch to clear up the confusing names.